Osiris/Hacks/Misc.cpp

```diff
@@ -1081,7 +1081,7 @@ void Misc::voteRevealer(GameEvent& event) noexcept
     memory->clientMode->getHudChat()->printf(0, " \x0C\u2022Osiris\u2022 %c%s\x01 voted %c%s\x01", isLocal ? '\x01' : color, isLocal ? "You" : entity->getPlayerName().c_str(), color, votedYes ? "Yes" : "No");
 }
 
-void Misc::onVoteStart(const void*& data, int& size) noexcept
+void Misc::onVoteStart(const void* data, int size) noexcept
```

Comment:
got this error

![image](https://user-images.githubusercontent.com/53382214/127801315-da70df1b-8e7d-4917-9e37-2df81a1fe95f.png)